<launch>

<!-- sis_v_writer_section-->

<!--sis_v-1p85m2019-dev_writer.conf -->
  <node pkg = "necst-1p85m2019"
        name = "sis_l1_v-1p85m2019-dev_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v_cmd' to = '/1p85m2019/sis_l1/v_cmd'/>
    <remap from = '/dev/cpz340816/rsw/ch' to = '/dev/cpz340816/rsw0/ch1'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/sis_v-1p85m2019-dev_writer.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_l2_v-1p85m2019-dev_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v_cmd' to = '/1p85m2019/sis_l2/v_cmd'/>
    <remap from = '/dev/cpz340816/rsw/ch' to = '/dev/cpz340816/rsw0/ch2'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_writer.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r1_v-1p85m2019-dev_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v_cmd' to = '/1p85m2019/sis_r1/v_cmd'/>
    <remap from = '/dev/cpz340816/rsw/ch' to = '/dev/cpz340816/rsw0/ch3'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_writer.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r2_v-1p85m2019-dev_writer"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v_cmd' to = '/1p85m2019/sis_r2/v_cmd'/>
    <remap from = '/dev/cpz340816/rsw/ch' to = '/dev/cpz340816/rsw0/ch4'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_writer.conf" />

  </node>


<!--sis_iv_reader-->
<!--sis_i-1p85m2019-dev_reader.conf-->
  <node pkg = "necst-1p85m2019"
        name = "sis_l1_i-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/i' to = '/1p85m2019/sis_l1/i'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch1'/>
        <param name = "config_file"
               value = "$(find necst-1p85m2019)/node_conf/sis_i-1p85m2019-dev_reader.conf" />
  </node>


    <node pkg = "necst-1p85m2019"
          name = "sis_l2_i-1p85m2019-dev_reader"
          type = "relay.py">
      <remap from = '/1p85m2019/sis/i' to = '/1p85m2019/sis_l2/i'/>
      <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch3'/>
          <param name = "config_file"
                 value = "$(find necst-1p85m2019)/node_conf/sis_i-1p85m2019-dev_reader.conf" />
    </node>

    <node pkg = "necst-1p85m2019"
          name = "sis_r1_i-1p85m2019-dev_reader"
          type = "relay.py">
      <remap from = '/1p85m2019/sis/i' to = '/1p85m2019/sis_r1/i'/>
      <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch5'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/sis_i-1p85m2019-dev_reader.conf" />

    </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r2_i-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/i' to = '/1p85m2019/sis_r2/i'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch7'/>
        <param name = "config_file"
               value = "$(find necst-1p85m2019)/node_conf/sis_i-1p85m2019-dev_reader.conf" />

  </node>


  <!--sis_v-1p85m2019-dev_reader.conf-->

  <node pkg = "necst-1p85m2019"
        name = "sis_l1_v-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v' to = '/1p85m2019/sis_l1/v'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch2'/>
        <param name = "config_file"
               value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_reader.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_l2_v-1p85m2019-dev_reader.conf"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v' to = '/1p85m2019/sis_l2/v'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch4'/>
        <param name = "config_file"
               value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_reader.conf" />

  </node>

  <node pkg = "necst-1p85m2019"
        name = "sis_r1_i-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v' to = '/1p85m2019/sis_r1/v'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch6'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_reader.conf" />

  </node>


  <node pkg = "necst-1p85m2019"
        name = "sis_r2_v-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m2019/sis/v' to = '/1p85m2019/sis_r2/v'/>
    <remap from = '/dev/cpz3177/rsw/ch' to = '/dev/cpz3177/rsw0/ch8'/>
        <param name = "config_file"
               value = "$(find necst-1p85m2019)/node_conf/sis_v-1p85m2019-dev_reader.conf" />
  </node>


<!--power section-->
<!--if_power-1p85m2019-dev_reader.conf-->

  <node pkg = "necst-1p85m2019"
        name = "if_power-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m/if/power' to = '/1p85m/if_l1/power'/>
    <remap from = '/dev/ma24126a/__port__/power' to = '/dev/ma24126a/__port__/power'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/if_power-1p85m2019-dev_reader.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "if_power-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m/if/power' to = '/1p85m/if_l2/power'/>
    <remap from = '/dev/ma24126a/__port__/power' to = '/dev/ma24126a/__port__/power'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/if_power-1p85m2019-dev_reader.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "if_power-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m/if/power' to = '/1p85m/if_r1/power'/>
    <remap from = '/dev/ma24126a/__port__/power' to = '/dev/ma24126a/__port__/power'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/if_power-1p85m2019-dev_reader.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "if_power-1p85m2019-dev_reader"
        type = "relay.py">
    <remap from = '/1p85m/if/power' to = '/1p85m/if_r2/power'/>
    <remap from = '/dev/ma24126a/__port__/power' to = '/dev/ma24126a/__port__/power'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/if_power-1p85m2019-dev_reader.conf" />
  </node>

<!--temp section-->
<!--dewar_temp-1p85m2019-dev_reader.conf-->
  <node pkg = "necst-1p85m2019"
        name = "dewar_temp-1p85m2019-dev_reader"
        type = "relay.py">
        <remap from = '/1p85m/dewar/temp' to = '/1p85m/dewar/temp_4k'/>
        <remap from = '/dev/218/__IP__/ch' to = '/dev/218/__IP__/ch1'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/dewar_temp-1p85m2019-dev_reader.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "dewar_temp-1p85m2019-dev_reader"
        type = "relay.py">
        <remap from = '/1p85m/dewar/temp' to = '/1p85m/dewar/temp_40k'/>
        <remap from = '/dev/218/__IP__/ch' to = '/dev/218/__IP__/ch2'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/dewar_temp-1p85m2019-dev_reader.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "dewar_temp-1p85m2019-dev_reader"
        type = "relay.py">
        <remap from = '/1p85m/dewar/temp' to = '/1p85m/dewar/temp_300k'/>
        <remap from = '/dev/218/__IP__/ch' to = '/dev/218/__IP__/ch3'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/dewar_temp-1p85m2019-dev_reader.conf" />
  </node>


<!--sg section-->
<!--first_sg-1p85m2019-dev_freq_writer.conf-->
  <node pkg = "necst-1p85m2019"
        name = "first_sg1-1p85m2019-dev_freq_writer"
        type = "relay.py">
        <remap from = '/1p85m/1st_sg/f_cmd' to = '/1p85m/1st_sg1/f_cmd'/>
        <remap from = '/dev/fsw0020/__IP__/f_cmd' to = '/dev/fsw0020/__IP__/f_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/first_sg-1p85m2019-dev_freq_writer.conf" />
  </node>

  <node pkg = "necst-1p85m2019"
        name = "first_sg2-1p85m2019-dev_freq_writer"
        type = "relay.py">
        <remap from = '/1p85m/1st_sg/f_cmd' to = '/1p85m/1st_sg2/f_cmd'/>
        <remap from = '/dev/fsw0020/__IP__/f_cmd' to = '/dev/fsw0020/__IP__/f_cmd'/>
    <param name = "config_file"
           value = "$(find necst-1p85m2019)/node_conf/first_sg-1p85m2019-dev_freq_writer.conf" />
  </node>

<!--second_sg-1p85m2019-dev_freq_writer.conf-->
    <node pkg = "necst-1p85m2019"
          name = "second_sg1-1p85m2019-dev_freq_writer"
          type = "relay.py">
          <remap from = '/1p85m/2nd_sg/f_cmd' to = '/1p85m/2nd_sg1/f_cmd'/>
          <remap from = '/dev/fsw0010/__IP__/f_cmd' to = '/dev/fsw0010/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/second_sg-1p85m2019-dev_freq_writer.conf" />
    </node>

    <node pkg = "necst-1p85m2019"
          name = "first_sg2-1p85m2019-dev_freq_writer"
          type = "relay.py">
          <remap from = '/1p85m/2nd_sg/f_cmd' to = '/1p85m/2nd_sg2/f_cmd'/>
          <remap from = '/dev/fsw0010/__IP__/f_cmd' to = '/dev/fsw0010/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/second_sg-1p85m2019-dev_freq_writer.conf" />
    </node>


<!--first_sg-1p85m2019-dev_onoff_writer.conf-->
    <node pkg = "necst-1p85m2019"
          name = "first_sg1-1p85m2019-dev_freq_writer"
          type = "relay.py">
          <remap from = '/1p85m/1st_sg/onoff_cmd' to = '/1p85m/1st_sg1/onoff_cmd'/>
          <remap from = '/dev/fsw0020/__IP__/f_cmd' to = '/dev/fsw0020/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/first_sg-1p85m2019-dev_onoff_writer.conf" />
    </node>

    <node pkg = "necst-1p85m2019"
          name = "first_sg2-1p85m2019-dev_freq_writer"
          type = "relay.py">
          <remap from = '/1p85m/1st_sg/onoff_cmd' to = '/1p85m/1st_sg2/onoff_cmd'/>
          <remap from = '/dev/fsw0020/__IP__/f_cmd' to = '/dev/fsw0020/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/first_sg-1p85m2019-dev_onoff_writer.conf" />
    </node>

<!--second_sg-1p85m2019-dev_onoff_writer.conf-->
    <node pkg = "necst-1p85m2019"
          name = "second_sg1-1p85m2019-dev_onoff_writer"
          type = "relay.py">
          <remap from = '/1p85m/2nd_sg/onoff_cmd' to = '/1p85m/2nd_sg1/onoff_cmd'/>
          <remap from = '/dev/fsw0010/__IP__/f_cmd' to = '/dev/fsw0010/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/second_sg-1p85m2019-dev_onoff_writer.conf" />
    </node>

    <node pkg = "necst-1p85m2019"
          name = "second_sg2-1p85m2019-dev_onoff_writer"
          type = "relay.py">
          <remap from = '/1p85m/2nd_sg/onoff_cmd' to = '/1p85m/2nd_sg2/onoff_cmd'/>
          <remap from = '/dev/fsw0010/__IP__/f_cmd' to = '/dev/fsw0010/__IP__/f_cmd'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/second_sg-1p85m2019-dev_onoff_writer.conf" />
    </node>


<!--vacuum section-->
<!--dewar_vacuum-1p85m2019-dev_reader.conf-->

    <node pkg = "necst-1p85m2019"
          name = "dewar_vacuum-1p85m2019-dev_reader"
          type = "relay.py">
          <remap from = '/dev/tpg/__usb__/press' to = '/dev/tpg/__usb__/press'/>
      <param name = "config_file"
             value = "$(find necst-1p85m2019)/node_conf/dewar_vacuum-1p85m2019-dev_reader.conf" />
    </node>

</launch>
